### ISSUE TITLE
Error Reading Snippet File

### ISSUE THREAD

**Issue by ghost** [amandeepmittal/atom-nodejs-snippets#2]
Package has an error reading the snippet file. This appears to be due to a duplicate entry for `process.nextTick()` on lines 83 and 431. 

If you require the stack trace for more information, I have put the output below.

```
<embedded>:15911 Error reading snippets file 'C:\Users\user\.atom\packages\atom-nodejs-snippets\snippets\nodejs-snippet.cson': Error: C:\Users\user\.atom\packages\atom-nodejs-snippets\snippets\nodejs-snippet.cson: Duplicate key 'process.nextTick()'
    at Object.detectDuplicateKeys (<embedded>:39452:19)
    at addProperty (<embedded>:170885:42)
    at Array.reduce (native)
    at Obj (<embedded>:170877:34)
    at transformNode (<embedded>:170953:18)
    at Value (<embedded>:170834:18)
    at transformNode (<embedded>:170953:18)
    at addProperty (<embedded>:170884:21)
    at Array.reduce (native)
    at Obj (<embedded>:170877:34)
    at transformNode (<embedded>:170953:18)
    at Value (<embedded>:170834:18)
    at transformNode (<embedded>:170953:18)
    at Block (<embedded>:170831:18)
    at transformNode (<embedded>:170953:18)
    at Object.parse (<embedded>:170959:22)
    at parseObject (<embedded>:39201:29)
    at parseContents (<embedded>:39261:22)
    at <embedded>:39381:30
    at FSReqWrap.oncomplete (fs.js:114:15)
```

**Comment by amandeepmittal** [comment]
Thanks! Let me verify it. Which Atom version are you using?

**Comment by ghost** [comment]
Currently using Beta, however, seems to also show up on my stable (1.25.1) build as well

**Comment by jayallen** [comment]
The problem is most certainly the second change in https://github.com/amandeepmittal/atom-nodejs-snippets/commit/e8e0f9d3fab7e3af45c2f25d37969d7c5bd2392d#diff-e6b56a5a0d1f5ac65a2c006c8b060f66R437

On my Mac, it's a duplicate key error. on `process.nextTick()`.  You have one on line 83 https://github.com/amandeepmittal/atom-nodejs-snippets/blob/e8e0f9d3fab7e3af45c2f25d37969d7c5bd2392d/snippets/nodejs-snippet.cson#L83-L89

And another on line 436 https://github.com/amandeepmittal/atom-nodejs-snippets/blob/e8e0f9d3fab7e3af45c2f25d37969d7c5bd2392d/snippets/nodejs-snippet.cson#L436-L443

I'd send you a pull request but I'm not sure whether you meant to get rid of one of them or give them different keys (e.g. `process.nextTick() es6`).

**Comment by jayallen** [comment]
Oops, I just noticed @ELuculent already said what the problem was.  Sorry. :)

**Comment by jayallen** [comment]
FWIW, I'm using Atom 1.25.1 on Mac and atom-nodejs-snippets v2.0.0

**Comment by amandeepmittal** [comment]
@ELuculent @jayallen I have fixed the issue in v2.1.1. I hope if you guys update it, you will not face the issue. Thus, I have closed this issue. If the problem occurs, please re-open the issue and I have removed the reference from line 83 to 89 to follow latest JavaScript coding patterns. Also, thank you so much for pointing it out 👍
